### ISSUE TITLE
[Bug]: <Remind me while Obsidian is closed?>

### ISSUE THREAD

**Issue by Pascal2810** [uphy/obsidian-reminder#97]
### Describe the bug

Is it possible that the reminders work while Obsidian is closed? Because right now they're only working while I have Obsidian opened.

### Steps to reproduce

description

### Operating system

Windows

**Comment by jakeparis** [comment]
Yes, I'd like this feature as well. It would greatly enhance the plugin.
